src/features/ExtensionCommands.ts

```diff
@@ -408,6 +418,36 @@ export class ExtensionCommandsFeature implements IFeature {
         return promise;
     }
 
+    private saveFile(filePath: string): Thenable<EditorOperationResponse> {
+
+        var promise: Thenable<EditorOperationResponse>;
+
+        // Make sure the file path is absolute
+        if (!path.win32.isAbsolute(filePath))
+        {
+            filePath = path.win32.resolve(
+                vscode.workspace.rootPath,
+                filePath);
+        }
+
+        // Normalize file path case for comparison
+        var normalizedFilePath = filePath.toLowerCase();
+
+        if (vscode.workspace.textDocuments.find(doc => doc.fileName.toLowerCase() == normalizedFilePath))
```

Comment:
Careful here.  On Linux, the file system is case-sensitive so this test could find the wrong file.  Probably need to do a runtime check of the OS or file-system.  I wonder if there is some function on node's OS or Path modules that could help?

Reply:
Very good point. I actually grabbed that from:
https://github.com/PowerShell/vscode-powershell/blob/master/src/features/ExtensionCommands.ts#L393

So I'll update that as well. Perhaps we should still normalize on Windows?

Reply:
Windows and I think macOS, right?  Doesn't it also have a case-insensitive file system?

Reply:
Ah. I didn't know macOS was case-insensitive! I'll fix the if-check tomorrow. It's late here so I'll get to it tomorrow.